configuration/java/http/README.md

```diff
@@ -13,10 +13,10 @@ This quickstart includes one service:
 ## Prerequisites
 
 - [Maven 3.x](https://maven.apache.org/install.html)
-- Java JDK 11 (or greater):
-  - [Microsoft JDK 11](https://docs.microsoft.com/en-us/java/openjdk/download#openjdk-11)
-  - [Oracle JDK 11](https://www.oracle.com/technetwork/java/javase/downloads/index.html#JDK11)
-  - [OpenJDK 11](https://jdk.java.net/11/)
+- Java JDK 17 (or greater):
+  - [Microsoft JDK 17](https://learn.microsoft.com/en-us/java/openjdk/download#openjdk-17)
```

Comment:
I was surprised Java 21 LTS is not the most used, but looking at State of java reports, those agree Java 17 is correct for right now.